mwm-5-dynamic-delivery/presentation/PITCHME.md

```diff
@@ -298,28 +298,66 @@ interface SplitInstallManager {
 
 ---
 
-### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Download</span><span style="text-transform: none; font-size:0.8em;"> a dynamic module</span>
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Install</span><span style="text-transform: none; font-size:0.8em;"> a dynamic module</span>
```

Comment:
Here we are presenting download + install-on-device as this method start both sequentially. The keyword `install` is maybe closer to what's done as it's wrapping the download process